build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -22,6 +22,135 @@ data:
     & antrea-agent --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
   Run-AntreaOVS-Containerd.ps1: |
     $ErrorActionPreference = "Stop"
+    $AntreaVersion = "3.0.5-antrea.0"
+    $OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$AntreaVersion-win64.zip"
+    $OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
```

Comment:
SHA256 hash is updated for 3.0.5 !

Reply:
Isn't this the same?
$OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'